packages/framework/request-handler/src/requestHandlers.ts

```diff
@@ -35,17 +35,19 @@ export const rootDataStoreRequestHandler = async (request: IRequest, runtime: IC
     const requestParser = RequestParser.create(request);
     const id = requestParser.pathParts[0];
     const wait = typeof request.headers?.wait === "boolean" ? request.headers.wait : undefined;
+    let rootDataStore;
```

Comment:
You can probably avoid the no-unsafe-return issue below by typing the `rootDataStore` here

```suggestion
    let rootDataStore: IFluidRouter;
```